refinery/user_files_manager/views.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from data_set_manager.utils import (search_solr, format_solr_response,
+                                    generate_solr_params_for_user)
+
+
+logger = logging.getLogger(__name__)
+
+
+class UserFiles(APIView):
+    def get(self, request):
+        params = request.query_params
+
+        solr_params = generate_solr_params_for_user(params, user=request.user)
```

Comment:
I might be missing something: `*.DoesNotExist` does not reflect a real error, so we can capture it and it won't reach this level. On the other hand, MultipleObjectsReturned does represent a real unexpected error, and letting this go all the way up and manifest as a 500 seems appropriate.